feature/google/src/test/java/com/revenuecat/purchases/google/StoreProductConversionsTest.kt

```diff
@@ -10,9 +11,26 @@ import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 class StoreProductConversionsTest {
+    @Test
+    fun `list of invalid ProductDetails with no price maps to empty list`() {
+        val productDetail1 = mockProductDetails(
+            productId = "iap_1",
+            type = BillingClient.ProductType.INAPP,
+            oneTimePurchaseOfferDetails = null,
+            subscriptionOfferDetails = null)
+        val productDetails = listOf(productDetail1)
+
+        val storeProducts = productDetails.toStoreProducts()
+        assertThat(storeProducts.size).isEqualTo(0)
```

Comment:
💯 wish there was some way we could send this error to the backend or something. to alert us that sometimes BC doesn't return a price..

Reply:
Agreeeeeeeeeeeed